pyopia/instrument/silcam.py

```diff
@@ -66,8 +66,13 @@ def load_bayer_rgb8(filename):
     array
         raw image float between 0-1
     '''
-    img_bayer = np.load(filename, allow_pickle=False)[:,:,0].astype(np.int16)
+    img_bayer = np.load(filename, allow_pickle=False).astype(np.int16)
+    if img_bayer.ndim == 3:
+        img_bayer = img_bayer[:,:,0]
```

Comment:
Could we do the same check as in mono8 loading, so this will raise an error if you accidentally give it an RGB image, for example?:

```python
image_shape = np.shape(im_mono)
    if len(image_shape) > 2:
        if image_shape[2] == 1:
            img = im_mono[:, :, 0]
        else:
            raise RuntimeError('Invalid image dimension')
    return img
```